src/KKManager.Core/KKManager.Core.csproj

```diff
@@ -13,43 +15,75 @@
     <FileAlignment>512</FileAlignment>
     <Deterministic>false</Deterministic>
     <TargetFrameworkProfile />
+    <NuGetPackageImportStamp>
+    </NuGetPackageImportStamp>
   </PropertyGroup>
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
     <DebugSymbols>true</DebugSymbols>
-    <DebugType>embedded</DebugType>
+    <DebugType>full</DebugType>
```

Comment:
All debug symbols are set to embedded because they work better in dnSpy, and any build scripts I make assume no pdb files get generated so they might begin to be included in releases.